tasks/extract-binaries-from-image/extract-binaries-from-image.yaml

```diff
@@ -83,7 +83,13 @@ spec:
           for DIGEST in $(jq -r ".layers[].digest" manifest.json)
           do
               FILE=${DIGEST#sha256:}
-              tar -xzvf "$FILE"
+              # Check if the archive contains the $IMAGE_PATH dir
+              if tar -tf "$FILE" | grep -q '^$IMAGE_PATH/'; then
+                echo "Extracting $FILE..."
+                tar -xzvf "$FILE"
```

Comment:
Since we already know the path is there, we can also just extract that path, no? i.e. `tar -xzvf "$FILE" "$IMAGE_PATH"`

Reply:
Nice catch! Fixed